gateway-service/src/chef/chef.service.ts

```
@@ -0,0 +1,89 @@
+        .catch((error) => observer.error(error));
+    });
+  }
+
+  async postSignUpChef(body: ChefModel[]): Promise<any> {
+    try {
+      const result = await lastValueFrom(
+        this.chefClient.send('chef/signup', body),
+      );
+      return result;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)